pennylane/tape/qnode.py

```diff
@@ -683,3 +683,40 @@ def qfunc_decorator(func):
         return update_wrapper(qn, func)
 
     return qfunc_decorator
+
+
+def make_drawing(_qnode, charset="unicode"):

```

Comment:
I know pylint will complain about this, but maybe it's better to just call this argument `qnode` anyway, so that it looks nicer to the user? Since the `qnode` decorator is not used internally by this method anyway, so it is okay to overwrite.

Alternatively, if we do
```python
def make_drawing(_qnode, charset="unicode"):
    """make_drawing(qnode, charset="unicode")
    Make a function that draws the given QNode.
```

we can overwrite the argument name in Sphinx.